Source/Scene/Cesium3DTile.js

```diff
@@ -297,28 +284,19 @@ define([
         this.selected = false;
```

Comment:
I think it's useful because it 's a flag that prevents us from double-selecting tiles when we do the traversal up to select a loaded ancestor.